examples/image_retraining/retrain.py

```diff
@@ -983,7 +983,7 @@ def logging_level_verbosity(logging_verbosity):
     return name_to_level[logging_verbosity]
   except Exception as e:
     raise RuntimeError('Not supported logs verbosity (%s). Use one of %s.' %
-                       (str(e), [*name_to_level]))
+                       (str(e), [name_to_level]))
```

Comment:
Oops, the original code works for Python 3 and prints all the keys, but it falls for Python 2 as you described.

Your code prints all the key-value pairs, but keys only is probably the better guidance.

Would you be fine with `list(name_to_level)`? Seems to work for me in Python 2.7 and 3.6.

Reply:
Yes, sure. Will you push this change directly to my PR branch?

Reply:
I'll fix it en route through our internal copy of the repo. (For tensorflow/hub, we merge pull requests into Google's revision control system and then sync them back out to GitHub. You will be credited as author, but technically, this pull request here will not show up as merged anyways.)

Thanks for bringing this up.